docs/content/configuration/indexing-service.md

```diff
@@ -83,6 +83,9 @@ The following configs only apply if the overlord is running in remote mode:
 |`druid.indexer.runner.taskCleanupTimeout`|How long to wait before failing a task after a middle manager is disconnected from Zookeeper.|PT15M|
 |`druid.indexer.runner.taskShutdownLinkTimeout`|How long to wait on a shutdown request to a middle manager before timing out|PT1M|
 |`druid.indexer.runner.pendingTasksRunnerNumThreads`|Number of threads to allocate pending-tasks to workers, must be at least 1.|1|
+|`druid.indexer.runner.maxRetriesBeforeBlacklist`|Number of times the middle manager can fail a task,  before it gets blacklisted, must be at least 1|5|
+|`druid.indexer.runner.taskBlackListBackoffTime`|How long to wait before a task is whitelisted again. This value should be greater that the value set for taskBlackListCleanupPeriod.|PT15M|
```

Comment:
rename this to workerBlackListBackoffTime & workerBlackListCleanupPeriod and modify docs to make it clear that we are blacklisting workers instead of tasks.